src/d3a/d3a_core/redis_connections/aggregator_connection.py

```diff
@@ -183,8 +183,12 @@ def consume_all_area_commands(self, area_uuid, strategy_method):
             response = [strategy_method({**command, 'transaction_id': transaction_id})
                         for command in area_commands]
             if transaction_id not in self.responses_batch_commands:
-                self.responses_batch_commands[transaction_id] = {aggregator_uuid: []}
-            self.responses_batch_commands[transaction_id][aggregator_uuid].append(response)
+                self.responses_batch_commands[transaction_id] = {aggregator_uuid: {area_uuid: []}}
```

Comment:
```suggestion
                self.responses_batch_commands[transaction_id] = {aggregator_uuid: {}}
```

As you already checking `area_uuid` one line below.